src/components/Carousel/Carousel.jsx

```diff
@@ -51,6 +51,7 @@ const Announcement = styled.div`
   overflow: hidden;
   clip: rect(1px, 1px, 1px, 1px);
```

Comment:
I believe clip is deprecated in favor of [clip-path](https://developer.mozilla.org/en-US/docs/Web/CSS/clip-path). Not a big deal, as I don't think browsers will drop support of it, but here's how it would look like:

```suggestion
  clip-path: inset(1px 1px 1px 1px);
```